pkgs/development/tools/continuous-integration/fly/default.nix

```diff
@@ -0,0 +1,40 @@
+{ buildGoModule, fetchFromGitHub, lib, writeText }:
+
+buildGoModule rec {
+  pname = "fly";
+  version = "5.3.0";
+
+  src = fetchFromGitHub {
+    owner = "concourse";
+    repo = "concourse";
+    rev = "v${version}";
+    sha256 = "06ns98k47nafhkkj7gkmxp7msn4ssypyss6ll0fm6380vq2cavnj";
+  };
+
+  modSha256 = "11rnlmn5hp9nsgkmd716dsjmkr273035j9gzfhjxjsfpiax60i0a";
+
+  subPackages = [ "fly" ];
+
+  buildFlagsArray = ''
+    -ldflags=
+      -X github.com/concourse/concourse.Version=${version}
+  '';
+
+  # The fly.bash file included with this derivation can be replaced by a
+  # call to `fly completion bash` once the `completion` subcommand has
+  # made it into a release. Similarly, `fly completion zsh` will provide
+  # zsh completions. https://github.com/concourse/concourse/pull/4012
+  postInstall = ''
+    install -D -m 444 ${
+      writeText "fly.bash" (builtins.readFile ./fly.bash)
+    } $out/share/bash-completion/completions/fly
+  '';
+
+  meta = with lib; {
+    description = "A command line interface to Concourse CI";
+    homepage = https://concourse-ci.org;
+    license = "Apache-2.0";
+    maintainers = with maintainers; [ ivanbrennan ];
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
List concatenating these two seems like an odd thing to do

Reply:
That's typical for the platforms. However, unless you really want to avoid this package on other platforms (such as Windows), I would leave this completely out. `buildGoModule` defaults it to `platforms.all`: https://github.com/NixOS/nixpkgs/blob/1ad69896552d88afc6f41cc7f0ddc392d8fff5fd/pkgs/development/go-modules/generic/default.nix#L196

Reply:
I went with the platforms I was able to test on, but I'll go with the `platforms.all` suggestion and remove this.